.github/workflows/hubble-cli.yaml

```diff
@@ -0,0 +1,147 @@
+name: Hubble CLI tests
```

Comment:
I think we need to add the new hubble-cli workflow to the [ariane-config workflow](https://github.com/cilium/cilium/blob/67164a81724ba803899359a9281c907e97e10f70/.github/ariane-config.yaml), potentially with its own target (e.g. `/ci-hubble-cli`)
```suggestion
name: Hubble CLI tests (ci-hubble-cli)
```